src/encoded/static/components/experiment.js

```diff
@@ -236,7 +236,11 @@ const LibraryProperties = ({ replicates }) => {
                         const strandSpecificityFirstValue = strandSpecificValues[0];
 
                         if (strandSpecificValues.every((value) => value === strandSpecificityFirstValue)) {
-                            strandSpecificity = `Strand-specific${strandSpecificityFirstValue === 'strand-specific' ? '' : ` (${strandSpecificityFirstValue})`}`;
+                            if (strandSpecificityFirstValue === 'unstranded') {
+                                strandSpecificity = strandSpecificityFirstValue;
+                            } else {
+                                strandSpecificity = `Strand-specific${strandSpecificityFirstValue === 'strand-specific' ? '' : ` (${strandSpecificityFirstValue})`}`;
+                            }
```

Comment:
So I think you could write this a little more concisely:

strandSpecificity = strandSpecificityFirstValue === 'unstranded' ? strandSpecificityFirstValue : `Strand-specific${strandSpecificityFirstValue === 'strand-specific' ? '' : ` (${strandSpecificityFirstValue})`}`;

That said, sometimes writing things this way is less readable, so that's the benefit to using if/else statements.

I think the "`" characters messed up how this is displaying so let me know if this isn't clear.

Reply:
I got eslint errors without the "`" characters so I had to add them back in.